upload/views/bundle_analysis.py

```diff
@@ -142,4 +144,29 @@ def post(self, request):
                 is_shelter_request=self.is_shelter_request(),
             ),
         )
+
+        if settings.TIMESERIES_ENABLED:
```

Comment:
do we want to jump right to creating or do we want to create if it doesn't already exist / isn't backfilled? the latter is what is done for coverage i think https://github.com/codecov/codecov-api/blob/66ec9fa93603d062868618a140fe6c154dc74d6f/timeseries/helpers.py#L341-L348

Reply:
I think its better to create it as soon as possible then it is to create it when we fetch the measurements, that way we don't miss out on a datapoint in the general flow of upload->process->query since the process step is when we insert the measurement datapoints which relies on the presence of the dataset rows being inserted.

With the coverage measurements its ok because when querying and not having dataset created it can still compute and return measurements on the fly and then do a backfill. Also with BA we don't plan on doing backfills because the previous bundle schemas don't support it.

Reply:
oh on second read the code is `get_or_create()`, i thought it was just `create()`. my concern was that you would create multiple `Dataset`s with the same type/repoID which sounded wrong. but this is all good